File: Linguagens_Programação/C/1.c
//1. Dado um inteiro x, determine 𝑥!
#include <stdio.h>

int main()
{
    int X; 
    int resultado = 1;
    printf("Digite um inteiro, o resultado será o seu fatorial: ");
    scanf("%i", &X);
    for (int i=1; i<=X ; i++){
        resultado = resultado * i;
        
    }
    printf("O resultado de %i fatorial é: %i",X,resultado);
    return 0;
}

File: Linguagens_Programação/Go/1.go
/*1. Dados um inteiro x e um inteiro não-negativo n, calcular 𝑥
𝑛. */
package main
import "fmt"

func main() {
    var x,n,total int
    
    fmt.Println("Digite um inteiro X: ")
    fmt.Scanln(&x)
    fmt.Println("Digite um inteiro N: ")
    fmt.Scanln(&n)
    total = 1
    for i := 1; i <= n; i++ {
        total *= x
    }
    fmt.Printf("Resultado de %d elevado a %d é: %d\n", x, n, total)
}